File: tinyQuest.server/Models/Hero.cs
namespace tinyQuest.Models
{
    public class Hero
    {
         public Hero()
      {
      }

      public int Id { get; set; }
      public string creatorId {get;set;}
      public string name { get; set; }
      public int raceId {get;set;}
      public int careerId {get;set;}
      public int health {get;set;}
      public int rangePower {get;set;}
      public int magicPower {get;set;}
      public int swordPower {get;set;}
      public Profile Creator{get;set;}
    }
}

File: tinyQuest.client/src/Models/Hero.js
export class Hero {
  constructor(data = {}) {
    this.id = data.id || data._id
    this.creatorId = data.creatorId
    this.raceId = data.raceId
    this.careerId = data.careerId
    this.health = data.health
    this.rangePower = data.rangePower
    this.magicPower = data.magicPower
    this.swordPower = data.swordPower
  }
}
